### ISSUE TITLE
Drop off sea lice samples for resequencing

### ISSUE THREAD

**Issue by shellytrigg** [RobertsLab/resources#940]

**Comment by shellytrigg** [comment]
:( the index adapters I used for these libraries are not compatible when only two samples are run because the indexes are too similar. 👎 

So I'll need to either:
1. remake one library and just sequence one sample 
2. remake both libraries from gDNA and use compatible adapters (e.g. 6 and 12) 
3. make a couple new libraries from sea lice in ethanol (need to isolate gDNA first)
    a. could pool lice and make 1 library from the pool 
    b. could make pools from the two populations and make 1 library from each pool (total 2 libraries)
    c. could make two libraries from 2 individuals from the same population
    d. could make two libraries from 1 individual from the each population
    e. maybe this is a calling to attempt low input MBD after all...?

@sr320 thoughts?

**Comment by sr320** [comment]
Why is there not an option - sequence one library? why would one need remaking?

**Comment by shellytrigg** [comment]
true! i'm clearly overthinking... we can sequence either library then. Should I just go with the one that got the most reads in the last run Sealice_F2 ?

**Comment by sr320** [comment]
Need to confirm they can bill by June 20.

**Comment by shellytrigg** [comment]
Erica at the Center says they can bill by June 20th no problem. also checking to see if we can just pay for the quote now and drop the samples off later

**Comment by shellytrigg** [comment]
an update from Peter at the Center: "We are allowed to invoice after we have  done the work, so as long as your samples are here a few days before June 20th, we can definitely get the invoice to you in time."

**Comment by sr320** [comment]
We need to do a check on non CG methylation in bismark to first make sure we are confident on BS treatment.

**Comment by shellytrigg** [comment]
with the alignment score min set to L,0,-0.6, here's the breakdown:

**Context**|**F1**|**F2**
:-----:|:-----:|:-----:
%mCpG|1.1|1
%mCHG|1|0.8
%mCHH|1.4|1.5

- [F1 cytosine report](https://gannet.fish.washington.edu/metacarcinus/Salmo_Calig/analyses/20200427/TG_PE_Calig_Aligned/Sealice_F1_S20_R1_001_val_1_bismark_bt2_pe.deduplicated_splitting_report.txt)

- [F2 cytosine report](https://gannet.fish.washington.edu/metacarcinus/Salmo_Calig/analyses/20200427/TG_PE_Calig_Aligned/Sealice_F2_S22_R1_001_val_1_bismark_bt2_pe.deduplicated_splitting_report.txt)

In the past I had [played around with different alignement score parameters on subsetted reads](https://shellytrigg.github.io/146th-post/) and here's a report on how different score_mins affect cytosine methylation in different contexts [SeaLiceBmrkAlignementScoreComparison.html](https://htmlpreview.github.io/?https://github.com/shellytrigg/Salmon_sealice/blob/master/analyses/AlignmentScoreComparison/SeaLiceBmrkAlignementScoreComparison.html).

I did the same alignment score analysis on subsetted salmon libraries that were prepped alongside the sea lice libraries with the same mastermix so I'd imagine the conversion rate would be the same across salmon and sea lice samples. Here's the alignment score comparison report for the subsetted salmon data:  [BmrkAlignementScoreComparison.html](https://htmlpreview.github.io/?https://github.com/shellytrigg/Salmon_sealice/blob/master/analyses/AlignmentScoreComparison/BmrkAlignementScoreComparison.html)

Here's a summary of the reports:
**Sample**|**Context**|**L,0,-0.2**|**L,0,-0.6**
:-----:|:-----:|:-----:|:-----:
sea lice|%mCpG|1|1.4
sea lice|%mCHG|0.5|1.1
sea lice|%mCHH|1|1.5
salmon|%mCpG|78|77
salmon|%mCHG|0.7|1.5
salmon|%mCHH|1.3|4

**Comment by sr320** [comment]
Problem here is mCpG methylation is not above “noise”. Will need libraries with lamda.

**Comment by shellytrigg** [comment]
@ sr320 I agree that prepping the libraries with lambda would be more convincing. However, it is possible that the BS conversion rate may still be around 99% making our mCpGs still within the margin of error. In looking through [this bisulfite seq bee paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6740151), their lambda BS conversion was ~99.3% and their genome-wide CpG methylation was 0.2-1%. They used [this script](https://github.com/BrendelGroup/BWASP/blob/master/bin/filterMsam.pl) to determine a reads threshold for confidently calling CpG methylation. The script has this comment about it:

```
This script will take as input a combined alignment/methylation call file in SAM format,
such as produced by Bismark, for either paired-end reads or single reads.  The script
first determines the probabilities of Z, X, and H calls.  A second pass through the
input determines read pairs (or single reads) that have a very small probability of
being properly bisulfite-treated reads - in the extreme, reads that produce methylation
calls in all possible C positions, independent of context.  The tacit assumption is that
the most parsimonious explanation of the data is that these reads represent sequence
evidence of DNA not properly exposed to the bisulfite-treatment.  Motivation for this
script came from analysis of social insect methylomes that have invariably shown very
low methylation rates (typically less than 1%).
```

Maybe we could do something like that or even follow the [BWASP workflow](https://github.com/BrendelGroup/BWASP)?
